STYLE.md

```diff
@@ -0,0 +1,48 @@
+# Style Guide
+
+This guide contains general guidelines for how to format and present documentation in this repository.
+They should be followed for most cases, but as a guideline it can be broken, _with good reason_.
+
+The documentation is written using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+
+## General Formatting
+
+- Format paragraphs with one sentence per line for easier diffs.
+- Leave a space before and after headings.
+
+## Headings are Written in Title Case
```

Comment:
This seems to be the case for most headings, but is not completely consistent. My purely subjective opinion is sentence case for headings as well, but don't care too much as long as we stick to one style.

Reply:
I've been doing and suggesting the opposite... But I also do not care as long as we are consistent.

Reply:
Personally I am more comfortable with lower case if I had to choose (except for the first word in the heading).

Of course, I am almost certainly wildly inconsistent in practice.

Reply:
https://github.com/eth-cscs/cscs-docs/pull/53/commits/dce66c6957cc6d9cbf08bd20eea423c99c824418